src/main/java/org/hydev/mcpm/utils/ZstdException.java

```diff
@@ -0,0 +1,22 @@
+package org.hydev.mcpm.utils;
+
+import java.io.IOException;
+
+/**
+ * Errors during the zstd compression/decompression
+ *
+ * @author Azalea (https://github.com/hykilpikonna)
+ * @since 2022-11-22
+ */
+public class ZstdException extends IOException
+{
+    public ZstdException(Throwable cause)
```

Comment:
Not sure where this is used but I'm curious why we won't use `rethrow` for something like this?

Reply:
Because we're using two different Zstd implementations here, and I want one exception to catch both of them